### PR TITLE
OCPQE-11339: fix imagepullbackoff issue when testing eventrouter

### PR DESCRIPTION
**Pull request by QiaolingTang**
To fix failures in [OCPQE-11339](https://issues.redhat.com//browse/OCPQE-11339)

### CODE REVIEW

**Comment by QiaolingTang** [comment]
Log: https://url.corp.redhat.com/52fe73b

File: features/step_definitions/logging.rb [hunk 1]
```diff
@@ -480,28 +480,29 @@
...
-    registry = image_content_source_policy('brew-registry').mirror_repository[0]
-    if image_version.start_with?("5")
-      # from logging 5.0, the image namespace is changed to openshift-logging
-      image = "#{registry}/rh-osbs/openshift-logging-#{image_name}:v#{image_version}"
-    else
-      image = "#{registry}/rh-osbs/openshift-ose-#{image_name}:v#{image_version}"
-    end
+  image_version = ""
+  # for logging 5.2 and later, use image tag v0.3.0/v0.4.0
+  if (clo_version.include? "5.5") || (clo_version.include? "5.4")
```
Comment (anpingli) [inline features/step_definitions/logging.rb:493]: Could we include 5.6?

Comment (QiaolingTang) [reply features/step_definitions/logging.rb:493]: OK

**Review by anpingli — COMMENTED** [review]
/lgtm

**Pull request merged**